### PR TITLE
16 many to many relations

### PR DESCRIPTION
**Pull request by nicolas-bastien**

### CODE REVIEW

**Comment by nicolas-bastien** [comment]
@mathieu-ducrot I made the test for the loader and it's working like this.
Do my test validate your use case in TARO ?

**Comment by nicolas-bastien** [comment]
@mathieu-ducrot I refactored the testing class as the same time I did the csv many to many so your exception test are used by the csv now ;-)

File: tests/Loader/DoctrineInsertUpdateLoaderTest.php [hunk 1]
```diff
@@ -76,6 +78,14 @@ function ($e) {
...
             ]
         );
+        $loader->addEntityToProcess(
+            Tag::class,
+            function ($e) {
+                return $e->getImportId();
+            },
+            'importId',
+            ['name']
+        );
```
Comment (mathieu-ducrot) [inline tests/Loader/DoctrineInsertUpdateLoaderTest.php]: @nicolas-bastien you need to add a test about the processing of the tags

**Comment by mathieu-ducrot** [comment]
@nicolas-bastien the many to many relations correctly work as expected on TARO. Can you make a release so i can test it on integration ?

**Pull request merged**